src/components/StorageOverview/Events/tests/__snapshots__/Events.test.js.snap

```diff
@@ -22,9 +22,7 @@ exports[`<Events /> renders correctly 1`] = `
     id="events-body"
     isLoading={false}
   >
-    <EventsBody>
-      <Component />
-    </EventsBody>
+    <EventsBody />
```

Comment:
I guess this is not the flow you intended to test. Can you please modify the tests to pass relevant data?